File: Operacoes/segundos.cpp
/*@segundos: Converter segundos em h:m:s*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");

    int tempo;
    cin >> tempo;

    int hora = tempo / 3600;
    int resto = tempo % 3600;
    int minutos = resto / 60;
    int segundos = resto % 60;

    cout << hora << ":" << minutos << ":" << segundos << "\n";

    return 0;
}

File: src/Operacoes/Segundos.java
/**
 * Questão: @segundos: Converter segundos em h:m:s
 *
 * @author UxieGu1
 */
package Operacoes;
import java.util.Scanner;

public class Segundos {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int tempo = scanner.nextInt();

        int hora = tempo / 3600;
        int resto = tempo % 3600;
        int minutos = resto / 60;
        int segundos = resto % 60;

        System.out.printf("%d:%d:%d%n", hora, minutos, segundos);
        scanner.close();
    }
}
